libs/@local/graph/authorization/src/api.rs

```diff
@@ -136,7 +136,7 @@ pub trait AuthorizationApi: Send + Sync {
     ) -> impl Future<Output = Result<Vec<ActorIdOrPublic>, Report<ReadError>>> + Send;
 
     fn modify_entity_relations(
-        &mut self,
+        &self,
```

Comment:
`mut` was not necessary because it's an external call, it was just used in case we want to implement another API without external services. Anyway, `mut self` was annoying when reusing the Postgres store. As we're going to remove it again, just making it non-mut makes the live a bit easier.